x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/helpers.test.tsx

```diff
@@ -0,0 +1,302 @@
+        const expected = {
+          isNew: false,
+          index: ['auditbeat-*'],
+          queryBar: {
+            query: {
+              query: '',
+              language: 'kuery',
+            },
+            filters: [],
+            saved_id: null,
+          },
+        };
+
+        expect(result.defineRuleData).toEqual(expected);
+      });
+    });
+
+    describe('aboutRuleData', () => {
+      test('returns timeline id and title of null if they do not exist on rule', () => {
+        const mockedRule = mockRuleWithEverything('test-id');
+        delete mockedRule.timeline_id;
+        delete mockedRule.timeline_title;
+        const result: GetStepsData = getStepsData({ rule: mockedRule });
+
+        expect(result.aboutRuleData?.timeline.id).toBeNull();
+        expect(result.aboutRuleData?.timeline.title).toBeNull();
+      });
+
+      test('returns name, description, and note as empty string if detailsView is true', () => {
+        const result: GetStepsData = getStepsData({
+          rule: mockRuleWithEverything('test-id'),
+          detailsView: true,
+        });
+
+        expect(result.aboutRuleData?.name).toEqual('');
+        expect(result.aboutRuleData?.description).toEqual('');
+        expect(result.aboutRuleData?.note).toEqual('');
+      });
+
+      test('returns note as empty string if property does not exist on rule', () => {
+        const mockedRule = mockRuleWithEverything('test-id');
+        delete mockedRule.note;
+        const result: GetStepsData = getStepsData({
+          rule: mockedRule,
+        });
+
+        expect(result.aboutRuleData?.note).toEqual('');
+      });
+    });
+
+    describe('scheduleRuleData', () => {
+      test('returns from as seconds if from duration is less than a minute', () => {
+        const mockedRule = {
+          ...mockRule('test-id'),
+          from: 'now-62s',
+          interval: '1m',
+        };
+        const result = getStepsData({ rule: mockedRule });
+
+        expect(result.scheduleRuleData?.from).toEqual('2s');
+        expect(result.scheduleRuleData?.interval).toEqual('1m');
+      });
+
+      test('returns from as minutes if from duration is less than an hour', () => {
+        const mockedRule = {
+          ...mockRule('test-id'),
+          from: 'now-660s',
+        };
+        const result = getStepsData({ rule: mockedRule });
+
+        expect(result.scheduleRuleData?.from).toEqual('6m');
+        expect(result.scheduleRuleData?.interval).toEqual('5m');
+      });
+
+      test('returns from as hours if from duration is more than 60 minutes', () => {
+        const mockedRule = {
+          ...mockRule('test-id'),
+          from: 'now-7400s',
+          interval: '5m',
+        };
+        const result = getStepsData({ rule: mockedRule });
+
+        expect(result.scheduleRuleData?.from).toEqual('1h');
+        expect(result.scheduleRuleData?.interval).toEqual('5m');
+      });
+
+      // Note: is this desired behavior?
```

Comment:
Is this a question you want to keep in the code or do you need to "@" someone here to get an answer?

Reply:
They were meant to remind me of things to follow up with Xavier on. Went ahead and removed just so I don't mistakenly forget to remove them beforehand.